colossus/src/main/scala/colossus/service/TagDecorator.scala

```diff
@@ -0,0 +1,17 @@
+package colossus
+package service
+
+import metrics.TagMap
+
+trait TagDecorator[I,O] {
```

Comment:
Good point.  I already have another branch in progress to add default decorators for existing codecs, I'll add some docs there.